README.md

```diff
@@ -159,6 +159,27 @@ Then just run it:
 $ PRONTO_GITLAB_API_PRIVATE_TOKEN=token pronto run -f gitlab -c origin/master
 ```
 
+***note: this requires at least Gitlab 11.x+**
+
+Merge request integration:
+
+```sh
+$ PRONTO_GITLAB_API_PRIVATE_TOKEN=token PRONTO_PULL_REQUEST_ID=id pronto run -f gitlab_mr -c origin/master
+```
+
+You can run it also on Gitlab CI:
+
+```sh
+pronto:
+  stage: code_review
+  script:
+    - apt-get install jq
+    - export PR_BRANCH=$(curl -s "https://${YOUR_GITLAB_URL}/api/v4/projects/${CI_PROJECT_ID}/merge_requests?private_token=${PRONTO_GITLAB_API_PRIVATE_TOKEN}&state=opened" | jq -r ".[]|select(.sha == \"$CI_COMMIT_SHA\")|.source_branch")
+    - export TR_BRANCH=$(curl -s "https://${YOUR_GITLAB_URL}/api/v4/projects/${CI_PROJECT_ID}/merge_requests?private_token=${PRONTO_GITLAB_API_PRIVATE_TOKEN}&state=opened" | jq -r ".[]|select(.sha == \"$CI_COMMIT_SHA\")|.target_branch")
+    - export PRONTO_PULL_REQUEST_ID=$(curl -s "https://${YOUR_GITLAB_URL}/api/v4/projects/${CI_PROJECT_ID}/merge_requests?private_token=${PRONTO_GITLAB_API_PRIVATE_TOKEN}&state=opened" | jq -r ".[]|select(.sha == \"$CI_COMMIT_SHA\")|.iid")
```

Comment:
We were also trying to do stuff like that (finding out which MR a job belongs to), all approaches have drawbacks that at least should be known. One thing which caught us by surprise is pagination (!). When you have more than 20 open MRs, the `merge_requests` API starts to paginate, and some MRs can simply not be found by this code here.

This can be an issue with long-running MRs. 

Also, force-pushing to a branch while / before the pronto job is run leads to the MR not being found. 

We did, unfortunately not find a reliable solution yet, so this is more a "note this" thing than a code review.

Anyways: great work on tackling this!

Reply:
Thanks for the heads up!

> One thing which caught us by surprise is pagination (!). When you have more than 20 open MRs, the `merge_requests` API starts to paginate, and some MRs can simply not be found by this code here.

The [logic used in the code](https://github.com/prontolabs/pronto/pull/308/files#diff-ea6e770489f3dd0b74bf42203b0b13c8R85) should not have that problem as it's auto paginating the merge request responses.



> Also, force-pushing to a branch while / before the pronto job is run leads to the MR not being found.

The failure only happens in the job that is running for the previous commit, right? I would assume that the job of the force pushed commit finds the MR again.

Reply:
Yes, these were just the issues that bit us when we used such code for review app stuff. But judging from the code, this part of the readme is outdated now and the `PRONTO_PULL_REQUEST_ID` env variable not needed?

Reply:
I've tried that but it does not work for me... (pronto was not able to find PR from branch)